ideas/310-core.md

```diff
@@ -52,23 +53,11 @@ Weekly sync-up calls
 - goals for the week
 - focus area, progress
 
-Each project that is currently active should have an umbrella issue that should
+Each project that is currently active should have an umbrella Epic in Pivotal Tracker that should
```

Comment:
Is this Wrike now?

Reply:
I didn't put it there, because we are still in a trial now. As soon as Wrike is official, I'll update.